packages/ffe-datepicker-react/src/datepicker/Datepicker.js

```diff
@@ -250,9 +253,10 @@ export default class Datepicker extends Component {
     closeCalendarSetInputFocus() {
         this.removeGlobalEventListeners();
         this.setState({
-            openOnFocus: false,
+            openOnFocus: true,
             displayDatePicker: false,
-        });
+            closeAfter: true
+        },this.dateInputRef._input.focus());
```

Comment:
`this.setState`'s second argument is a function. Pass in an arrow function to make sure `react` can call it whenever it has flushed its state changes:

```js
this.setState(
  {
    openOnFocus: true,
    displayDatePicker: false,
    closeAfter: true,
  }, 
  () => this.dateInputRef._input.focus(),
);
```

Reply:
Thanks, committed the change now.